src/js/common/schemaform/helpers.js

```diff
@@ -204,6 +204,24 @@ export function filterInactivePages(pages, form) {
   }, form.data);
 }
 
+export function stringifyForm(key, value) {
```

Comment:
Mmm...I think this name is misleading since it doesn't actually stringify the form, but is used in the stringification process. Maybe something more like `cullFields`...or something.

But then, I'm no good at naming; I already feel sorry for my unborn children.